uiowa/uiowa
# [Wellbeing] Remove display of new taxonomy terms

The new taxonomy terms (Audience, Topic, Type) ended up getting included in the page display.

It's been confirmed that these are not to be displayed, and should be for back-end use only.





# How to test


```
ddev blt ds --site=wellbeing.uiowa.edu && ddev drush @wellbeing.local uli node/add/page
```

Check for a clean sync and config import.
Add some terms (audience, topic, and type) and save the page
See that they are not displayed on the page
